app/models/protected_area.rb

```diff
@@ -147,6 +147,18 @@ def overlap(pa)
     overlap
   end
 
+  def self.global_terrestrial_oecm_coverage
+    land_oecms = ProtectedArea.where(is_oecm: true, marine: false).pluck(:reported_area)
```

Comment:
@stanleypliu Here and also in the method below, you could make use of the scope already implemented in this class instead of the where statement, so you can also concatenate them.
e.g.

```
self.oecms.marine_areas.pluck(...)
```